packages/components/Button/src/ButtonTokens.android.ts

```diff
@@ -3,76 +3,86 @@ import { globalTokens } from '@fluentui-react-native/theme-tokens';
-        },
-        focused: {
-          paddingHorizontal: globalTokens.spacing.m,
-        },
+        borderWidth: globalTokens.stroke.width20,
+        borderInnerWidth: globalTokens.stroke.width10,
       },
     },
-    small: {
-      padding: globalTokens.spacing.xs - globalTokens.stroke.width10,
+    outline: {
       borderWidth: globalTokens.stroke.width10,
-      iconSize: 16,
-      focused: {
-        borderWidth: 0,
-        padding: globalTokens.spacing.xs,
+      disabled: {
+        borderWidth: globalTokens.stroke.width10,
+      },
+      pressed: {
+        borderWidth: globalTokens.stroke.width10,
       },
-      hasContent: {
-        minWidth: 64,
-        paddingHorizontal: globalTokens.spacing.s - globalTokens.stroke.width10,
-        hasIconAfter: {
-          spacingIconContentAfter: globalTokens.spacing.xs,
-        },
-        hasIconBefore: {
-          spacingIconContentBefore: globalTokens.spacing.xs,
-        },
-        focused: {
-          paddingHorizontal: globalTokens.spacing.s,
-        },
+      focused: {
+        borderWidth: globalTokens.stroke.width20,
+        borderInnerWidth: globalTokens.stroke.width10,
       },
     },
+    block: {
+      width: '100%',
+    },
     large: {
-      padding: globalTokens.spacing.s - globalTokens.stroke.width10,
-      borderWidth: globalTokens.stroke.width10,
+      paddingHorizontal: globalTokens.spacing.l,
+      paddingVertical: globalTokens.spacing.s,
+      borderRadius: globalTokens.corner.radius60,
+      borderWidth: 0,
       iconSize: 20,
-      focused: {
-        borderWidth: 0,
-        padding: globalTokens.spacing.s,
+      outline: {
+        borderWidth: globalTokens.stroke.width10,
+        iconSize: 20,
+      },
+      fontSize: t.typography.variants.body1Strong.size,
+      fontFamily: t.typography.variants.body1Strong.face,
+      fontWeight: t.typography.variants.body1Strong.weight,
+      spacingIconContentBefore: globalTokens.spacing.xs,
+      spacingIconContentAfter: globalTokens.spacing.xs,
+      minHeight: 36,
+      minWidth: 36,
+    },
+    medium: {
+      paddingHorizontal: globalTokens.spacing.s,
+      paddingVertical: globalTokens.spacing.xs,
+      borderRadius: globalTokens.corner.radius40,
+      borderWidth: 0,
+      outline: {
+        borderWidth: globalTokens.stroke.width10,
       },
-      hasContent: {
-        minWidth: 96,
-        paddingHorizontal: globalTokens.spacing.l - globalTokens.stroke.width10,
-        hasIconAfter: {
-          spacingIconContentAfter: globalTokens.spacing.sNudge,
-        },
-        hasIconBefore: {
-          spacingIconContentBefore: globalTokens.spacing.sNudge,
-        },
-        focused: {
-          paddingHorizontal: globalTokens.spacing.l,
-        },
+      iconSize: 20,
+      fontSize: t.typography.variants.body2Strong.size,
+      fontFamily: t.typography.variants.body2Strong.face,
+      fontWeight: t.typography.variants.body2Strong.weight,
```

Comment:
Button has a ButtonFontTokens file which these would belong to

Reply:
Added